build.sbt

```diff
@@ -19,7 +19,7 @@ import UnidocKeys._
 
 name := "pio"
 
-version in ThisBuild := "0.9.7-SNAPSHOT"
+version in ThisBuild := "0.10.0-snapshot"
```

Comment:
This is causing integration tests to fail. `snapshot` needs to be uppercase.

RE:
https://github.com/apache/incubator-predictionio/blob/develop/tests/docker-files/init.sh#L27
https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN401